src/main/java/org/fundacionjala/at15/spaceinvaders/Block.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.at15.spaceinvaders;
+
+import static org.fundacionjala.at15.spaceinvaders.Constants.Block.*;
+import java.awt.Graphics;
+import java.awt.Color;
+
+public class Block {
+    private int lifeAsteroid;
+
+    public Block(int life) {
+        this.lifeAsteroid = life;
```

Comment:
should be asteroidLife

Reply:
Thank you! It's a grammar error, I'll correct it.